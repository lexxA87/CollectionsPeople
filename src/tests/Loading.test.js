import Loading from "../components/helper/Loading";
import ShallowRenderer from "react-test-renderer/shallow";

it("renders correctly Loading component", () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Loading />);
  const view = renderer.getRenderOutput();

  expect(view).toMatchSnapshot();
});
