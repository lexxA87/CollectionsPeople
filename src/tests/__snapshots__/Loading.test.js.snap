// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Loading component 1`] = `
<Spinner
  animation="border"
  role="status"
>
  <span
    className="visually-hidden"
  >
    Loading...
  </span>
</Spinner>
`;

exports[`renders correctly react-test-renderer 1`] = `
<Spinner
  animation="border"
  role="status"
>
  <span
    className="visually-hidden"
  >
    Loading...
  </span>
</Spinner>
`;
